### PR TITLE
Terrain Recognition using RESNET50

### PR DESCRIPTION
**Pull request by ramana2074**
Pretrained models have already learned to recognize general features and patterns in data, which can improve their performance on your specific task.

### CODE REVIEW

**Comment by AKSHITHA-CHILUKA** [comment]
@Akasxh despite of many times telling you have not changed the labels.
Please add labels 
The crt labels are level3 with no spaces , gssoc-ext is the label not gssoc24-extd , hactoberfest-accepted is the label not hactoberfest .
I hope you will change all the label's in all the issues and pr's as soon as possible .

**Comment by Akasxh** [comment]
Could you please completely make a seperate folder with name of technique used just as transfer learning folder that has been made. Use similar architecture and make sure to include all those components in  them.

**Comment by ramana2074** [comment]
added into a residual learning file

**Comment by Akasxh** [comment]
Please add Plot training and validation accuracy over epochs. This is crucial to study how different techniques are working.

**Comment by ramana2074** [comment]
added Plots

**Comment by Akasxh** [comment]
Try making your model more efficient from what I've see your accuracy is pretty low.
image

**Comment by ramana2074** [comment]
terrain output
is this okay?

**Comment by Akasxh** [comment]
Sorry to say it's not yet fully optimised try adding some multimodality of resnets itself to see how they turn out to be.
